### PR TITLE
Fix: TreeBuilder Constructor for Symfony 5.1

### LINKED ISSUE #12: Install package on Symfony 5.1
Hi 

I have problem to install our package on Symfony 5.1 

When i composer require 

> Problem 1
>     - Root composer.json requires cethyworks/google-place-autocomplete-bundle ^5.1 -> satisfiable by cethyworks/google-place-autocomplete-bundle[v5.1].
>     - cethyworks/google-place-autocomplete-bundle v5.1 requires twig/twig ^1.0||^2.0|^5.0 -> found twig/twig[1.3.0, ..., v1.44.1, v2.0.0, ..., v2.14.1] but the package is fixed to v3.1.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.`

This error message is return

I test to put directly on composer.json and 

> HTTP 500 Internal Server Error
> Too few arguments to function Symfony\Component\Config\Definition\Builder\TreeBuilder::__construct(), 0 passed in /vagrant/source/public/AzAgency/vendor/cethyworks/google-place-autocomplete-bundle/src/DependencyInjection/Configuration.php on line 20 and at least 1 expected

This error is return 

Thx

### PR DESCRIPTION
**Pull request by aniskasmi**
Fix #12 

Modification of treeBuilder for Symfony 5.1 remove ->root() and replace on __construct of TreeBuilder

### CODE REVIEW

**Comment by aniskasmi** [comment]
@Cethy I make some modification for Treebuilder

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -17,8 +17,8 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('cethyworks_google_place_autocomplete');
+        $treeBuilder = new TreeBuilder( 'cethyworks_google_place_autocomplete' );
+        $rootNode = $treeBuilder;
```
Comment (Cethy) [inline src/DependencyInjection/Configuration.php]: According to [this](https://symfony.com/blog/new-in-symfony-4-2-important-deprecations#deprecated-tree-builders-without-root-nodes), it needs `->getRootNode()` to work.
Did you try it in this state ?

**Comment by aniskasmi** [comment]
@Cethy I realy want this PR

**Review by Cethy — CHANGES_REQUESTED** [review]

**Comment by aniskasmi** [comment]
@Cethy i update and add getRootNode()

**Pull request merged**